esm/interface/css-style-declaration.js

```diff
@@ -13,11 +13,10 @@ const updateKeys = style => {
     if (attr) {
       style.set(PRIVATE, attr);
       for (const rule of attr[VALUE].split(/\s*;\s*/)) {
-        const pair = rule.split(/\s*:\s*/);
-        if (1 < pair.length) {
-          let [key, value] = pair;
+        const [key, ...rest] = rule.split(':');
```

Comment:
in two lines we have `key = key.trim()` which is meant to be there because:

```js
document.documentElement.style.cssText = 'background-color : white';
```

I hence believe this `const [...]` should be a `let [...]` as it was before, or the tests fail and nothing lands.

Reply:
Of course! Sorry about that. Should be fixed.